project/LibraryVersions.scala

```diff
@@ -1,13 +1,13 @@
 
 object LibraryVersions {
-  val circeVersion = "0.12.1"
+  val circeVersion = "0.14.1"
   val awsClientVersion = "1.11.827"
   val catsVersion = "2.1.1"
   val jacksonVersion = "2.10.5"
-  val jacksonDatabindVersion = "2.10.5.1"
+  val jacksonDatabindVersion = "2.11.4"
```

Comment:
now we are pulling in 2.11 anyway, so pin the version with no vulnerability https://snyk.io/vuln/maven:com.fasterxml.jackson.core%3Ajackson-databind